### DISCUSSION TITLE
My fish cursor is missing

### DISCUSSION THREAD

**Discussion by davidshare** [fish-shell/fish-shell#9146] [Q&A] [answered]
Hi I recently installed fish on my Ubuntu 20.04 OS, and it runs well except the cursor is missing. It doesn't display so I am not aware of where the next Character will be displayed. 

When I activate fish on my other shell environment - terminator, it works well. But the fish shell I installed doesn't work.

Please how do I fix this?

![Screenshot from 2022-08-17 22-48-16](https://user-images.githubusercontent.com/25446108/185249268-29d08b18-0095-411f-ac64-e862bb95723d.png)

**Comment by faho** [comment, marked as answer]
This is a problem in your terminal and needs to be fixed there. Fish isn't involved in the actual drawing and doesn't turn the cursor off, so there's nothing it can do.

Either find the right setting, report a bug to whoever makes your terminal (which probably isn't a great idea because it's likely an older version, because you're running an LTS distribution), or switch to a different one.
